### PR TITLE
[browser] hide fireperimeter and add flag to show hidden props

### PR DESCRIPTION
**Pull request by beets**
append `?all=1` to the url to show all hidden properties like fireperimeter, geojsoncoordinates, etc

/cc @ajaits

### CODE REVIEW

File: static/js/browser/arc_table_row.tsx [hunk 1]
```diff
@@ -123,14 +123,16 @@ export class ArcTableRow extends React.Component<
...
+        {this.props.provenanceId && this.props.src ? (
           <td className="provenance-column">
             {this.props.provenanceId && (
               <a href={HREF_PREFIX + this.props.provenanceId}>
                 {this.props.src}
               </a>
             )}
           </td>
+        ) : (
+          <td></td>
```
Comment (beets) [inline static/js/browser/arc_table_row.tsx:135]: this fixes a DOM validation warning

File: static/js/browser/out_arc_section.tsx [hunk 2]
```diff
@@ -64,6 +66,9 @@ export class OutArcSection extends React.Component<
   OutArcSectionPropType,
   OutArcSectionStateType
 > {
+  // Whether to show all properties
+  showAll: boolean;
```
Comment (chejennifer) [inline static/js/browser/out_arc_section.tsx:70]: I don't think this is used?

**Review by chejennifer — APPROVED** [review]
nice, this would be useful!

File: static/js/browser/out_arc_section.tsx (hunk 2 above)
Comment (beets) [reply static/js/browser/out_arc_section.tsx]: ah, thanks for catching that!

**Pull request merged**